cloudinit/net/eni.py

```diff
@@ -421,6 +422,11 @@ def _render_route(self, route, indent=""):
         return content
 
     def _render_iface(self, iface, render_hwaddress=False):
+        iface = copy.deepcopy(iface)
+
+        # Remove irrelevant keys
+        with suppress(KeyError):
```

Comment:
I like using `suppress`, but in this case it doesn't appear to add value. Wouldn't something like this suffice?
```python
if iface.get("config_id"):
    iface.pop("config_id")
```